### ISSUE TITLE
Feature Request: Specify dimensions of a tile window with resizewindowpixel

### ISSUE THREAD

**Issue by v4u6h4n** [hyprwm/Hyprland#5739]
### Description

Hey there :-)

After some testing it seems that `resizewindowpixel` can only apply dimensions to floating windows, as mentioned in #5710. I would like to be able to apply specific dimensions to tiled windows as well. I'm working on an experimental configuration for twitch streaming with my GF, where instead of using scenes in OBS, I'm just changing 'scenes' with hyprctl scripts, and controlling the exact dimensions of the web cam tile would really simplify things.

**Comment by guttermonk** [comment]
I agree - would be cool if I could specify that mpv opens to 25% width in tiled mode or something like that.

**Comment by vaxerski** [comment]
Hello there!

This issue has been closed, as we are moving from issues to discussions. More about this can be found [here](https://wiki.hyprland.org/Contributing-and-Debugging/Issue-Guidelines/) and in #9854.

Firstly, please make sure this issue is still relevant in latest Hyprland. If so, we ask you to open a [discussion](https://github.com/hyprwm/Hyprland/discussions) (please read the discussion guidelines first, in the pinned post)

Thank you for your understanding!
